packages/rmb_direct_client/lib/types/lib/types.ts

```diff
@@ -1,6 +1,6 @@
 /**
  * Generated by the protoc-gen-ts.  DO NOT EDIT!
- * compiler version: 3.20.3
+ * compiler version: 3.21.12
```

Comment:
` DO NOT EDIT! ` :D
 
 install the required version and run the `build.sh` instead if you want to update the version in docstring here

Reply:
This version is updated automatically according to the version of protoc I'm using (the same version Azmy is using). The "DO NOT EDIT" warning pertains to proto-gen-ts, I believe. Additionally, if the new version successfully generates the types, then we are ready to proceed.